src/App.jsx

```diff
@@ -284,7 +287,25 @@ const MainContent = ({
   page,
   setPage,
   scrollTarget,
+  totalElements,
 }) => {
+  /**
+   * function to calculate and show correct results for relevant page.
+   *
+   * @param {number} resultsLength - the amount of results per page.
+   *
+   * @return {string} - returns a string with the amount of results
+   */
+  const amountOfResults = resultsLength => {
```

Comment:
Might be worth moving this method to the `utils` folder. This feels like a method that would be used for other areas of the site, for instance if we added a section to the detail view that fetched other services from the same organisation we might use pagination there. 

Its one of those totally subjective things!

`import { amountOfResults } from './lib/utils'`